src/Flysystem/LocalFilesystemAdapter.php

```diff
@@ -19,6 +33,8 @@ public function __construct(
         ?MimeTypeDetector $mimeTypeDetector = null
     ) {
         $this->pathPrefixer = new PathPrefixer($location);
+        $this->linkHandling = $linkHandling;
+        $this->visibility = $visibility ??= new PortableVisibilityConverter();
```

Comment:
See parent class constructor - though since that sets private properties we have to do it here instead to access them